_docker/nginx/default.conf

```diff
@@ -0,0 +1,22 @@
+server {
+    listen 80;
+    index index.php index.html;
+    server_name localhost;
+    error_log  /var/log/nginx/error.log;
+    access_log /var/log/nginx/access.log;
+    root /slackemon;
+
+    location ~ /\. {
+        deny all;
+    }
+
+    location ~ \.php$ {
+        try_files $uri =404;
+        fastcgi_split_path_info ^(.+\.php)(/.+)$;
+        fastcgi_pass php:9000;
```

Comment:
`php:9000` is the address to forward .php requests. `php` is the hostname of php container since Docker Compose automatically sets hostnames (https://docs.docker.com/compose/networking/)